src/sqlancer/transformations/SimplifyConstant.java

```diff
@@ -0,0 +1,105 @@
+package sqlancer.transformations;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import net.sf.jsqlparser.expression.DoubleValue;
+import net.sf.jsqlparser.expression.Expression;
+import net.sf.jsqlparser.expression.LongValue;
+import net.sf.jsqlparser.expression.StringValue;
+import net.sf.jsqlparser.statement.StatementVisitorAdapter;
+import net.sf.jsqlparser.statement.insert.Insert;
+import net.sf.jsqlparser.statement.select.Select;
+import net.sf.jsqlparser.util.deparser.ExpressionDeParser;
+import net.sf.jsqlparser.util.deparser.SelectDeParser;
+
+/**
+ * Shorten the constant of a statement e.g. "a_very_long_str" -> "", 12341234->1;
+ * <p>
+ * Note: Since the API of JSQLParser may have some problems with double values: when `setValue` is called, it doesn't
+ * change the string of the statement, double values are not handled here.
```

Comment:
not sure if I understand the comment here. how do double values relate to string values?

Reply:
In JSQLParser, The `DoubleValue` class contains two members: `String stringValue` and `double value`. When `setValue` is called, It just modifies `value` while `getString()` returns `stringValue`. When the code below is executed:
``` java
DoubleValue doubleValue = new DoubleValue("2.0");
doubleValue.setValue(1.0);
System.out.println(doubleValue);
```
The result is still `2.0`.  For more details, we can refer to `DoubleValue.class`

While for `LongValue` objects, its StringValue could be modified by calling `setStringValue`;